pandas/tests/test_expressions.py

```diff
@@ -21,13 +21,7 @@
 
 
 if not expr._USE_NUMEXPR:
-    try:
-        import numexpr  # noqa
-    except ImportError:
-        msg = "don't have"
-    else:
-        msg = "not using"
-    raise nose.SkipTest("{0} numexpr".format(msg))
+    numexpr = pytest.importorskip('numexpr')
```

Comment:
yeah this is the only way to actually skip the entire module (well you *can* simply use a skipif on the classes, but this is cleaner).